src/transforms/addEslintDisableComment.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Type definitions for VSCode autocompletion!
+ *
+ * @typedef {Object} TransformParams
+ * @property {*} ast - The resulting AST as parsed by babel
+ * @property {import("ast-types/gen/builders").builders} builder - Recast builder for transforming the AST
+ * @property {*} options - Options passed into the transform from the CLI (if any)
+ */
+
+/**
+ * @param {TransformParams} param0
+ * @returns {import("ast-types").Visitor}
+ */
+
+const reactHookPrefix = new RegExp("use\\w+");
+const transform = ({ builder }) => {
+  return {
+    visitExportNamedDeclaration(path) {
+      const includesHook = path.node.specifiers.some(specifier => {
+        if (reactHookPrefix.test(specifier.local.name)) return true;
+      });
+
+      if (includesHook) {
+        // create exact copy of the export declaration
+        const newDeclaration = builder.exportNamedDeclaration(path.node.declaration, [
+          ...path.node.specifiers,
+        ]);
+        const newDeclarations = path.node.specifiers.map((_, index) => {
```

Comment:
Simply inserting the node by `path.node.specifiers = ...` gives a type error because it's a comment block, not declaration block. so I had to make a copy and create a new array, then replace the original path. 

Referred to [this comment](https://github.com/benjamn/recast/issues/181#issue-81583133) as an idea

Reply:
Wow, nice find Asuka :) interesting solution, but it does make sense.